projects/admin/src/app/acquisition/routes/guards/can-budget-active.guard.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2022 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router } from '@angular/router';
+import { RecordService } from '@rero/ng-core';
+import { BaseApi } from '@rero/shared';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CanBudgetActiveGuard implements CanActivate {
```

Comment:
Reflexion : Don't know if the class name sounds good ; maybe "isBudgetActiveGuard" ? I suggest, you choose ;-)